### ISSUE TITLE
/public/AssetPairs would be nice.

### ISSUE THREAD

**Issue by andyDoucette** [btschwertfeger/python-kraken-sdk#90]
Many apps will need at some point to find out how to refer to specific currencies in a kraken-API-friendly (with all the Z's, X's, etc).

I've found the /public/AssetPairs endpoint to be helpful for that, as it maps the asset names to the pair names without having to rely on a concatenation mapping that is ill-defined in reverse. (Is "ABCDEFG" "ABC/DEFG" or "ABCD"/"EFG" or "A"/"BCDEFG"?)

I'm using this currently:

```
from functools import lru_cache
import requests

@lru_cache(maxsize=100)
def tradeable_asset_pairs_get():
    'Gets information about all of the currency conversions we can do.'
    
    response=requests.get('https://api.kraken.com/0/public/AssetPairs')
    assert response.status_code ==200, 'Error getting tradeable asset pairs.'
    
    return response.json()
```

I imagine other people might want this endpoint too.  Can it be added to the SDK please?

**Comment by btschwertfeger** [comment]
Hello @andyDoucette!

Thank you for mentioning this issue! This endpoint is already implemented in the python-kraken-sdk v1.2.0: [python-kraken-sdk/blob/v1.2.0/kraken/spot/market/__init__.py#L108-L174](https://github.com/btschwertfeger/python-kraken-sdk/blob/v1.2.0/kraken/spot/market/__init__.py#L108-L174] but it is only callable using the `pair` parameter. 

You are right, this endpoint can be called without any parameter. I will change the `pair` parameter to be optional.

**Comment by andyDoucette** [comment]
Great, thank you. :)
